### PR TITLE
MWPW-129482 New Marketo UI

### PR DESCRIPTION
**Pull request by meganthecoder**
* New Marketo UI (intended for MCZ POC)
* Move cleaning functions to Marketo product

Resolves: [MWPW-129482](https://jira.corp.adobe.com/browse/MWPW-129482)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/methomas/marketo-ui?martech=off
- After: https://methomas-marketo-ui--milo--adobecom.hlx.page/drafts/methomas/marketo-ui?martech=off

### CODE REVIEW

File: libs/blocks/marketo/marketo.js [hunk 1]
```diff
@@ -156,13 +119,16 @@ const init = (el) => {
         formWrapper.append(description);
       }
 
-      const marketoForm = createTag('form', { ID: `mktoForm_${formID}`, class: 'hide-errors' });
-      formWrapper.append(marketoForm);
-      fragment.append(error, formWrapper);
+      const marketoForm = createTag('form', { ID: `mktoForm_${formID}`, class: 'hide-errors my-form', style: 'opacity:0;visibility:hidden;' });
```
Comment (Brandon32) [inline libs/blocks/marketo/marketo.js:122]: What's `my-form`?

**Review by Brandon32 — APPROVED** [review]
Looks good!

File: libs/blocks/marketo/marketo.js (hunk 1 above)
Comment (meganthecoder) [reply libs/blocks/marketo/marketo.js]: Good catch! That was from testing, so I've removed it

File: libs/blocks/marketo/marketo.js [hunk 2]
```diff
@@ -156,13 +119,16 @@ const init = (el) => {
         formWrapper.append(description);
       }
 
-      const marketoForm = createTag('form', { ID: `mktoForm_${formID}`, class: 'hide-errors' });
-      formWrapper.append(marketoForm);
-      fragment.append(error, formWrapper);
+      const marketoForm = createTag('form', { ID: `mktoForm_${formID}`, class: 'hide-errors', style: 'opacity:0;visibility:hidden;' });
+      const span1 = createTag('span', { id: 'mktoForms2BaseStyle', style: 'display:none;' });
```
Comment (honstar) [inline libs/blocks/marketo/marketo.js:123]: Just curious what these hidden `span1` and `span2` elements are for?

**Review by honstar — APPROVED** [review]

File: libs/blocks/marketo/marketo.js [hunk 3]
```diff
@@ -20,32 +20,10 @@ const HIDDEN_FIELDS = 'hidden fields';
...
-    }
-  });
-};
-
-const cleanFormStyles = (form) => {
-  const formEl = form.getFormElem().get(0);
-
-  formEl?.querySelectorAll('style').forEach((e) => { e.remove(); });
-  formEl?.parentElement?.querySelectorAll('*[style]').forEach((e) => e.removeAttribute('style'));
-};
```
Comment (JasonHowellSlavin) [inline libs/blocks/marketo/marketo.js:41]: A little surprised you've deleted this. Are we leveraging some of the styles sent with the forms? Just curious.

**Review by JasonHowellSlavin — APPROVED** [review]

File: libs/blocks/marketo/marketo.js (hunk 2 above)
Comment (meganthecoder) [reply libs/blocks/marketo/marketo.js:123]: Rob asked us to put these in, so they're helpers for code being loaded by the Marketo product

File: test/blocks/marketo/marketo.test.js [hunk 4]
```diff
@@ -21,76 +21,56 @@ describe('marketo', () => {
...
+    });
 
-    formValidate(form, false, error, errorMessage);
-    expect(error.classList.contains('alert')).to.be.true;
-  });
-
-  it('marketo field error visible', async () => {
-    const button = await waitForElement('.marketo button');
-    button.click();
+    it('marketo hidden fields', async function () {
```
Comment (JasonHowellSlavin) [inline test/blocks/marketo/marketo.test.js:40]: nit: `it` strings should start with a verb

File: test/blocks/marketo/marketo.test.js [hunk 5]
```diff
@@ -21,76 +21,56 @@ describe('marketo', () => {
...
+      formValidate(form);
 
-  it('validate marketo fields success', async () => {
-    const error = await waitForElement('.marketo-error');
-    const errorMessage = 'There are some fields that require your attention';
+      expect(formEl.classList.contains('show-warnings')).to.be.true;
+    });
 
-    expect(error).to.exist;
+    it('marketo field error visible', async () => {
```
Comment (JasonHowellSlavin) [inline test/blocks/marketo/marketo.test.js:56]: same here. I know these are not your strings, but just a thought.

**Review by JasonHowellSlavin — APPROVED** [review]

File: test/blocks/marketo/marketo.test.js (hunk 4 above)
Comment (meganthecoder) [reply test/blocks/marketo/marketo.test.js]: Done

**Review by JasonHowellSlavin — APPROVED** [review]

**Pull request merged**